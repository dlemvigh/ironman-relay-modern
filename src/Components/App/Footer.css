.footer {
  position: absolute;
  bottom: 0;
  background-color: #555d61;
  color: white;
  width: 100%;    
  margin-top: 2rem;
  padding-top: 1rem;
  padding-bottom: 1rem;
}

.footer a, .footer a:hover, .footer a:focus {
  color: white;
}
.footer a {
  text-decoration: underline;
}
.footer a:hover {
  text-decoration: none;
}


.footer .footer-left {
  float: left;
}

.footer .footer-right {
  float: right;
}