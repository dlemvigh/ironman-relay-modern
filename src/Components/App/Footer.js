import React from "react";
import { Container } from "reactstrap";
import './Footer.scss';

class Footer extends React.Component {
  render() {
    return (
      <footer className="footer">
        <Container>
          <div className="footer-left">
            <strong>Powered by Enthusiasm</strong>
          </div>
          <div className="footer-right">
            App icons by <a href="https://icons8.com/">Icons8</a>
          </div>
        </Container>
      </footer>
    )
  }
}

export default Footer;
