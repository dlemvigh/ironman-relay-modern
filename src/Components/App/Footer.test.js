import React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer";

it("renders without crashing", () => {
  shallow(<Footer />);
})

it("matches snapshot", () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
})