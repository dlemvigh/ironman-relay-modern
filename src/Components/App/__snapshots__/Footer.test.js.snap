// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<footer
  className="footer"
>
  <Container
    tag="div"
  >
    <div
      className="left"
    >
      <strong>
        Powered by Enthusiasm
      </strong>
    </div>
    <div
      className="right"
    >
      App icons by 
      <a
        href="https://icons8.com/"
      >
        Icons8
      </a>
    </div>
  </Container>
</footer>
`;
